### ISSUE TITLE
Custom message for array validation

### ISSUE THREAD

**Issue by ardyfeb** [poppinss/indicative#245]
How to set custom message for array validation ?, i have trying like below code but message still showing `required validation variant.0.value failed`

## Package version
5.0.8

## Node.js and npm version
latest

## Sample Code (to reproduce the issue)

```
data = {
   variant: [
     {
        value: "wkwkland"
     }
   ]
}

validate(data, {
  'variant.0.value': 'required'
}, {
  'variant.0.value.required': 'Variant value cannot be empty'
})
```

**Comment by thetutlage** [comment]
Shouldn't it be `variant.0.value.required`?

**Comment by ardyfeb** [comment]
> Shouldn't it be `variant.0.value.required`?

i edited my issue, actually is `variant` i typo when writing issue

**Comment by ardyfeb** [comment]
![Screenshot from 2019-07-15 17-48-36](https://user-images.githubusercontent.com/30074366/61211099-c8bef900-a728-11e9-84ee-5bc07e87e30f.png)
here a complete rules and  message

**Comment by thetutlage** [comment]
Works fine for me with this code sample.

```js
const  { validate } = require('indicative/validator')

const data = {
   variant: [{}]
}

validate(data, {
  'variant.0.value': 'required'
}, {
  'variant.0.value.required': 'Variant value cannot be empty'
}).catch(console.error)
```

**Comment by thetutlage** [comment]
Here's the live example https://repl.it/@amanvirk/Indicative-example

**Comment by ardyfeb** [comment]
thanks for your help, solved by upgrading from `5.0.8` to `7.2.0`
